src/main/io/displayport_msp_bf_compat.c

```diff
@@ -62,10 +62,10 @@ uint8_t getBfCharacter(uint8_t ch, uint8_t page)
     case SYM_DEGREES:
         return BF_SYM_GPS_DEGREE;
 
-/*
     case SYM_HEADING:
-        return BF_SYM_HEADING;
+        return BF_SYM_ARROW_NORTH;
```

Comment:
This is trying a bit too hard. The inav icon represents a compass, this is picking an arrow pointing up.

Reply:
@mmosca But BF do not have something looks like "Compass", so we may choose something like char 'C' or just arrow to represent following number is for heading.

Reply:
> @mmosca But BF do not have something looks like "Compass", so we may choose something like char 'C' or just arrow to represent following number is for heading.

There are a lot of symbols not included in BF. The goal for the BFCOMPAT modes is not to  have 0 question marks, but to map the symbols that betaflight actually supports and make it clear when a symbol is not supported, without adding too much complexity to the code, or ambiguity to the OSD.

Unfortunatelly, only DJI can actually fix the overall situation. :(